build.sbt

```diff
@@ -1,24 +1,24 @@
 lazy val commonSettings = Seq(
-  organization in ThisBuild := "com.eed3si9n"
+  ThisBuild / organization := "com.eed3si9n",
+  scalaVersion := "2.12.8"
 )
 
 lazy val root = (project in file(".")).
+  enablePlugins(SbtPlugin).
   settings(commonSettings: _*).
   settings(
     sbtPlugin := true,
-    crossSbtVersions := Vector("0.13.18", "1.2.8"),
     name := "sbt-unidoc",
     description := "sbt plugin to create a unified API document across projects",
     licenses := Seq("Apache License v2" -> url("http://www.apache.org/licenses/LICENSE-2.0.html")),
     scalacOptions := Seq("-deprecation", "-unchecked"),
-    publishArtifact in (Compile, packageBin) := true,
-    publishArtifact in (Test, packageBin) := false,
-    publishArtifact in (Compile, packageDoc) := false,
-    publishArtifact in (Compile, packageSrc) := true,
+    Compile / packageBin / publishArtifact := true,
+    Test / packageBin / publishArtifact := false,
+    Compile / packageDoc / publishArtifact := false,
+    Compile / packageSrc / publishArtifact := true,
```

Comment:
`publish / skip := true` should be sufficient, it isn't necessary anymore to do all the disabling individually

Reply:
haha oops